src/snowflake/connector/network.py

```diff
@@ -679,6 +679,7 @@ def _get_request(
         headers: dict[str, str],
         token: str = None,
         timeout: int | None = None,
+        is_fetch_query_status: bool = False,
```

Comment:
I feel we also need this for other queries?

Reply:
what are the other queries you mention here? I know there are other GET requests used in the code base. but I prefer to doing this error handling carefully and case by case to avoid unexpected side affect.